src/api/app/models/scm_status_report.rb

```diff
@@ -0,0 +1,43 @@
+# This class represents the communication that took place between OBS and the
+# SCM to render each one of the check marks.
+#
+# Those check marks appear as green, orange and red ticks at the bottom of a
+# PR or next to a commit, on GitHub.
+#
+# The details of the request and the response will be shown in the
+# Reports to the SCM tab in Workflow Runs show page.
+class SCMStatusReport < ApplicationRecord
```

Comment:
```suggestion
class ScmStatusReport < ApplicationRecord
```

Reply:
Its not working for me with uppercase, I guess this is not possible with Models...

Reply:
I get this 
```
[f245a8f7-842b-497f-946a-e7d67df5ce51] ActionView::Template::Error (uninitialized constant WorkflowRun::ScmStatusReport

            raise NameError.new("uninitialized constant #{candidates.first}", candidates.first)
            ^^^^^):
[f245a8f7-842b-497f-946a-e7d67df5ce51]     2: .card
[f245a8f7-842b-497f-946a-e7d67df5ce51]     3:   .card-body
[f245a8f7-842b-497f-946a-e7d67df5ce51]     4:     = render(WorkflowRunHeaderComponent.new(workflow_run: @workflow_run))
[f245a8f7-842b-497f-946a-e7d67df5ce51]     5:     = render(WorkflowRunDetailComponent.new(workflow_run: @workflow_run))
[f245a8f7-842b-497f-946a-e7d67df5ce51]   
[f245a8f7-842b-497f-946a-e7d67df5ce51] app/components/workflow_run_detail_component.rb:15:in `initialize'
[f245a8f7-842b-497f-946a-e7d67df5ce51] app/views/webui/workflow_runs/show.html.haml:5:in `new'
[f245a8f7-842b-497f-946a-e7d67df5ce51] app/views/webui/workflow_runs/show.html.haml:5
[f245a8f7-842b-497f-946a-e7d67df5ce51] config/initializers/wrap_parameters.rb:37:in `call'
```

Reply:
The same on my side, I'll revert this inflection and fix it later